nginx_proxy/start_nginx.sh

```diff
@@ -32,25 +33,38 @@ $(command basename $0)
 (2) Starts nginx with a custom Endpoints service name and service version, and
 have nginx obtain the service configuration:
 $(command basename $0) -n ENDPOINTS_SERVICE_NAME -v ENDPOINTS_SERVICE_VERSION
+(3) Starts nginx with a custom Endpoints service name and managed rollout
+strategy
+$(command basename $0) -n ENDPOINTS_SERVICE_NAME -r managed
 Options:
     -h
         Shows this message.
     -n ENDPOINTS_SERVICE_NAME
-        Required. The name of the Endpoints Service.
+        Optional. The name of the Endpoints Service.
```

Comment:
Please specify that other arguments will be ignored when using ENDPOINTS_SERVICE_NAME.

Reply:
The other arguments are not ignored when service name is given. It's actually the opposite. When service name is not given, then the other args (version and rollout strategy) are ignored.

Agreed that this behaviour should be documented though.

Reply:
Added comments